plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinition.tsx

```diff
@@ -76,6 +76,8 @@ const useStyles = makeStyles((theme: BackstageTheme) => ({
       ...theme.typography.button,
       borderRadius: theme.shape.borderRadius,
       color: theme.palette.primary.main,
+      // override whatever may be in the theme's typography to ensure consistency with asyncapi
+      'text-transform': 'inherit',
```

Comment:
```suggestion
      textTransform: 'inherit',
```

this is the react native form of writing it